api/HTMLAnchorElement.json

```diff
@@ -239,54 +239,6 @@
           }
         }
       },
-      "media": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLAnchorElement/media",
```

Comment:
https://developer.mozilla.org/docs/Web/API/HTMLAnchorElement/media is 404 and I've already cleaned up https://developer.mozilla.org/docs/Web/API/HTMLAnchorElement